torch/_dynamo/compiled_autograd.py

```diff
@@ -354,6 +458,10 @@ def end_capture(self, outputs):
             (self.fx_tracer.create_arg(self.to_proxy(outputs)),),
             {},
         )
+        runtime_inputs_to_move: List[int] = []
+        if snapshot_cudagraph_enabled():
+            runtime_inputs_to_move = self.move_graph_nodes_to_cuda(self.fx_tracer.graph)
+        # TODO: remove the graph node's dummy metadata
```

Comment:
@xmfan what is this?

Reply:
it's dangerous to do graph passes over this initial capture graph at the moment, there's incorrect tensor metadata attached to the nodes

Reply:
but it doesn't affect current passes

Reply:
I left the TODO there (and reworded it a bit)